core/services/offchainreporting/delegate.go

```diff
@@ -171,13 +181,15 @@ func (d Delegate) ServicesForSpec(jobSpec job.SpecDB) (services []job.Service, e
 		contractTransmitter := NewOCRContractTransmitter(concreteSpec.ContractAddress.Address(), contractCaller, contractABI,
 			NewTransmitter(db, ta.Address(), d.config.EthGasLimitDefault(), d.config.EthMaxUnconfirmedTransactions()))
 
+		runResults := make(chan pipeline.RunWithResults, d.config.JobPipelineResultWriteQueueDepth())
```

Comment:
I would have probably been ok with hardcoding this one in a const somewhere, but now its here I won't complain